BusinessLayer/SongPlayer/SongPlayer.cpp

```diff
@@ -124,6 +127,11 @@ void SongPlayer::updateInfo()
 {
     artist_ = mediaPlayer_.metaData(QMediaMetaData::ContributingArtist).toString();
     title_ = mediaPlayer_.metaData(QMediaMetaData::Title).toString();
-    cover_ = mediaPlayer_.metaData(QMediaMetaData::CoverArtImage).value<QImage>();
-    emit updateGUI(title_, artist_);
+    album_ = mediaPlayer_.metaData(QMediaMetaData::AlbumTitle).toString();
+    album_.replace(" ", "");
```

Comment:
A comment here to explain this line would be appreciated

Reply:
Also, what happens if the album name is multiple words?

Reply:
Multiple word in the album name won't cause any problems when opening the jpg files. I set both the file name and the album name retrieved from QT to contain no spaces so its easier to access the file when loading images.